builds-cleaner.groovy

```diff
@@ -0,0 +1,30 @@
+/*** BEGIN META {
+  "name" : "Builds Cleaner",
+  "comment" : "Delete all builds keeping ${MAX_BUILDS} from all job of the Jenkins Instance. It checks build is not building and it also keeps the keep forever, lastStableBuild, lastSuccessfulBuild, lastSuccessfulBuild, lastUnstableBuild, lastUnsuccessfulBuild in case are not included in the ${MAX_BUILDS}.",
+  "parameters" : [],
+  "core": "2.107.3",
+  "authors" : [
+    { name : "Marco Davalos" }
+  ]
+} END META**/
```

Comment:
You don't mention a license.  I would like to keep the license of this project consistent.  Will you allow your work to fall under the [MIT license defined in this project](https://github.com/samrocketman/jenkins-script-console-scripts/blob/2fd0a4eaedaa30e8157fa0b0c564d0a2a942689e/LICENSE.txt)?

Also, this project is not scriptler.  I recommend you following the same convention as the rest of the scripts in this repository where a license is defined at the top and then a description of the script follows.  This immediately allows others to see what your script is licensed under.  It also easily satisfies the MIT requirement to present the license when somebody copies your work.  For an example, see the top header comments of any script in this repository.

I tend to update the license comments every year, if you don't want that burden then you don't have to define an end date.

Example defining an end date:
```
Copyright (c) 2015-2018 Sam Gleske - https://github.com/samrocketman/jenkins-script-console-scripts
```

Example, leaving the copyright grant perpetually open ended.

```
Copyright (c) 2015-, Sam Gleske - https://github.com/samrocketman/jenkins-script-console-scripts
```

# This comment is the only required change

You are only required to make this change.  If you don't want to change the other parts then that's fine.  I will change them myself later because I will be following your license grant.

Basically the rest of the feedback in this PR is optional for you.